.github/workflows/test.yml

```
@@ -20,6 +20,10 @@ jobs:
           cache: 'npm'
       - name: Install dependencies
         run: npm install
+      - name: Fetch ENV from .env.ci
+        uses: ./.github/actions/dotenv-action
+        with:
+          path: .env.ci
```

this would only work if we would include that action within this repo, however, the way easier approach would be the following:

```suggestion
      - name: Fetch ENV from .env.ci
        run: |-
          cat .env.ci >> $GITHUB_ENV
```